tests/test_other.py

```diff
@@ -10409,6 +10409,12 @@ def test_supported_linker_flags(self):
     out = run_process([PYTHON, EMCC, path_from_root('tests', 'hello_world.cpp'), '-Xlinker', '--print-map'], stderr=PIPE).stderr
     self.assertContained('warning: ignoring unsupported linker flag: `--print-map`', out)
 
+    out = run_process([PYTHON, EMCC, path_from_root('tests', 'hello_world.cpp'), '-Wl,-rpath=foo'], stderr=PIPE).stderr
+    self.assertContained('warning: ignoring unsupported linker flag: `-rpath=foo`', out)
+
+    out = run_process([PYTHON, EMCC, path_from_root('tests', 'hello_world.cpp'), '-Wl,-rpath-link,foo'], stderr=PIPE).stderr
+    self.assertContained('warning: ignoring unsupported linker flag: `-rpath-link`', out)
```

Comment:
maybe a loop or parametrized test, as we have at least 3 almost identical pairs here?

Reply:
Meh.. maybe, I'm kind of loop-averse when it comes to test code.  It means that line numbers no longer uniquely identify failures.    I could parameterize it but.. meh.

Reply:
Fine with me either way.